userguide/kaui/includes/analytics.adoc

```diff
@@ -1,3 +1,9 @@
-// === Analytics and Reporting
+The Analytics page is only available when the analytics plugin has been deployed.
 
-// === Deposits (Wire Transfers)
+It lets you display custom metrics with filtering capabilities. We provide a few by default, but the analytics framework lets you add any desired metric.
+
+Here we show the MRR by plan over time:
+
+image::../assets/img/kaui/analytics-mrr.png[width=850,align="center"]
+
+For more information, see the https://docs.killbill.io/latest/userguide_analytics.html#components-analytics-overview[_Analytics Guide_].
```

Comment:
>> The userguide_kaui.pdf file does not include images  

Oh, wow. I made a mistake in generating it, obviously. I'll fix the image refs and regenerate the PDF.

Reply:
>>I was not able to review is [Getting-Started.adoc](https://github.com/FlowWriter/killbill-docs/blob/kaui-only-14/userguide/kaui/includes/getting-started.adoc).

"Getting Started" was renamed from "Navigation and Interface" (or whatever I called it since I started working on it).